jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -4,7 +4,7 @@
         # TODO(fejta): consider a stable tag instead of master
         git clone https://github.com/kubernetes/test-infra -b master
         './test-infra/jenkins/bootstrap.py' \
-            --bare \
+            '{soak-repos}' \
```

Comment:
I realize that you want `'--repo=k8s.io/kubernetes --repo=k8s.io/release'` to mean two flags, but highly confident that what this code does is define a single repo flag whose value is `k8s.io/kubernetes --repo=k8s.io/release`

But happy to see us answer this question by submitting the PR :grin: